packages/uxpin-merge-cli/src/resources/Button/Button.js

```diff
@@ -19,10 +18,4 @@ Button.propTypes = {
   label: PropTypes.node,
```

Comment:
Let's make this a string too.

```suggestion
  label: PropTypes.string,
```